src/components/file-upload/file-upload.yaml

```
@@ -11,6 +11,10 @@ params:
   type: string
   required: false
   description: Optional initial value of the input
+- name: describedBy
+  type: string
+  required: false
+  description: Optional text or element id to add to the `aria-describedby` attribute to provide a description for screenreader users.
```

Only element IDs will work with `aria-describedby` – text can't be passed. How about something like…

```suggestion
  description: One or more element IDs to add to the `aria-describedby` attribute, used to provide additional descriptive information for screenreader users.
```

?

---

Perfect, I've gone with this one in the latest push